src/config.ts

```diff
@@ -56,6 +56,12 @@ const validators = {
   REACT_APP_MIDGARD_URL: url({
     default: 'https://midgard.thorchain.info/v2',
   }),
+  REACT_APP_COSMOS_NODE: url({
+    default: 'https://rest.cosmos.directory/cosmoshub/',
```

Comment:
@toshiSat thanks for changing this - a directory seems more robust here, assuming they're all the same "variety" of nodes.

i know you said a handful of them you tried didn't work - is this the case that they're offline, or that it's similar to infura in that we need an archival node, and not just any kind of node? is there a specific flavor/tier of node that we need, and are we ensuring with these endpoints they're going to provide us those nodes that are guaranteed to work?

would you mind adding a small comment here for maintainers in future?

Reply:
We need a REST node that has CORS enabled.  A lot of them that I tried did not, such as our own hosted datahub nodes

Reply:
@0xdef1cafe How did you determine that a `.directory` domain is more robust?